Release 2.1.0: Merge development -> master

davidgrayston opened the pull request:
### Changed
- Button now opens QR in modal
- "Yoti Dashboard" is now referred to as "Yoti Hub"

### Fixed
- LICENSE in root now matches GNU LICENSE in ./yoti/ directory

emmas-yoti commented on docker/Dockerfile:
```diff
@@ -1,79 +0,0 @@
-FROM php:7.1-apache
```
Should there be something in the Release Notes about the rename of the Docker files?

davidgrayston replied:
These are only for local development, CI and demo purposes - @echarrod is it worth mentioning this?

echarrod replied:
I think we could add it to our internal dev process wiki, but no worth adding to the README if clients/public don't need to know about them

emmas-yoti commented on yoti/admin/views/yoti/tmpl/default.php:
```diff
@@ -8,14 +8,15 @@
-                <?php echo JText::_('You need to first create a Yoti App at'); ?> <a href="<?php echo \Yoti\YotiClient::DASHBOARD_URL; ?>" target="_blank">
-                    <?php echo JText::_('Yoti Dashboard'); ?>
+                <?php echo JText::_('You need to first create a Yoti App at'); ?> <a href="<?php echo YotiHelper::YOTI_HUB_URL; ?>" target="_blank">
+                    <?php echo JText::_('Yoti Hub'); ?>
```
Dashboard->Hub should probably be in the release notes?

davidgrayston replied:
Updated PR description

emmas-yoti commented on LICENSE:
This needs mentioning in the changelog

davidgrayston replied:
The license is already GNU here: https://extensions.joomla.org/extension/yoti/ so will omit there - I'll include in the changelog on GitHub release

davidgrayston replied:
Updated PR description

davidgrayston commented:
@echarrod I've updated the version for release in 2dee365307b0b8bad16e031f7bf01919d7caa5c9 - there was a duplicate `2.0.0` entry, which I've changed to `2.1.0` https://docs.joomla.org/Deploying_an_Update_Server#Extension

davidgrayston commented:
@echarrod `./yoti/admin/com_yoti.xml` is copied during installation but shouldn't be included in the packed plugin - fixed in cd8df75

>Note: This has only been an issue since installing from the mounted working copy of the extension.
